### ISSUE TITLE
Yalu nukes the sandbox

### ISSUE THREAD

**Issue by mologie** [kpwn/yalu102#228]
I'm pointing out the obvious here, and am aware that this is by design.

Previous jailbreaks came with specific sandbox patches. Yalu removes all restrictions, meaning that any app can read the data of any other app (i.e. Facebook gets access to your emails.) Nice for the hackers of us, not so nice for Josh from 7th grade who wants to install Pokemon Go tweaks, but does not realise what he's doing to his device.

Is a more selective sandbox patch planned?

**Comment by beanjo55** [comment]
its a beta, pretty sure the little things like this are not in yet.

**Comment by kpwn** [comment]
Pangu9.3.3 had the same patches.

**Comment by kpwn** [comment]
Anyway it is doable. But effort.

**Comment by mologie** [comment]
OK, thank you for the comments. I was thinking about the time where rocketbootstrap was required due to the original sandbox (or a part of it) still being active. I probably remembered iOS 7 / evasi0n. It's interesting to learn that Pangu just took the nuclear approach, too.

**Comment by kpwn** [comment]
I do not patch mach bootstrap server sandbox checks.

**Comment by shdkpr2008** [comment]
It's a opensource which means basically any app after open-sourcing yalu102 can re-use the code to do any thing possible, until and unless you won't run those kind of apps.

**Comment by iOS4all** [comment]
Can we use JIT functionality without patch kernel? Or the kernel is already patched?
It's important to patch kernel to remove all restrictions including JIT.

**Comment by beanjo55** [comment]
@iOS4all if he didnt answer when you an issue, why would he respond now? the code is here, look and see if it has what you need

**Comment by iOS4all** [comment]
@beanjo55 you're absolutely right. And I'm sorry. Unfortunately I didn't know anything about codes and something like that. I just want to said that creator of retroarch just said in order to use JIT on all emulation projects the creator of jailbreak most patch the kernel that's it. I just said that because this jailbreak is special because of fully KPP bypass like TaiG.
Thanks a lot.

**Comment by beanjo55** [comment]
@iOS4all then the creator of retroarch should ask or look.

**Comment by 0neday** [comment]
I think @kpwn need do more effort to fix sandbox patch without effect yalu102 app to execute
